packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -475,6 +477,19 @@ contract L2Genesis is Deployer {
         _setImplementationCode(Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER);
     }
 
+    /// @notice This predeploy is following the saftey invariant #1.
+    ///         This contract has no initializer.
+    function setETHLiquidity() internal {
+        _setImplementationCode(Predeploys.ETH_LIQUIDITY);
+        vm.deal(Predeploys.ETH_LIQUIDITY, type(uint248).max);
```

Comment:
Why is it set to `type(uint248).max` in genesis?  I don't find the rationale in the [spec](https://github.com/ethereum-optimism/design-docs/blob/391ae8ce540e69d9bfef9ec7e1e32c27c6878357/protocol/interoperable-ether.md).

Reply:
This is because it is large enough to allow arbitrary outflow without running risk of overflow on inflow. It would be really bad if a liquidity constraint resulted in the inability to send or receive ether between domains.

We should make sure the spec clearly includes this in https://github.com/ethereum-optimism/specs/pull/256

Reply:
Added this detail to the specs PR, thank you.